_studio/mfx_lib/shared/src/mfx_session.cpp

```diff
@@ -601,6 +601,29 @@ void _mfxSession::Clear(void)
 
 void _mfxSession::Cleanup(void)
 {
+    // wait untill all task are processed
```

Comment:
Small typo in a comment. 
I wonder if we should rearrange wait sequence. In a typical transcode with vpp scenario, decode tasks would be stopped first, then vpp, then encode. @dvrogozh, @artem-shaporenko, @eistarov do you see any point in sorting wait sequence?